### ISSUE TITLE
Add version parameter

### ISSUE THREAD

**Issue by camilamaia** [scanapi/github-action#3]
Add a parameter to receive the ScanAPI version

Example:
```
- uses: scanapi/github-action@v1
  with:
    scanapi-version: 2.0.0
    arguments: 'run ./scanapi.yaml'
```

**Comment by ShubhamSood1406** [comment]
Can I work on it? Please assign it to me.

**Comment by camilamaia** [comment]
@ShubhamSood1406 sure thing! Done ✅ 

Thanks!

**Comment by camilamaia** [comment]
Closed by #4
